src/mds/SessionMap.cc

```diff
@@ -681,6 +681,8 @@ void Session::decode(bufferlist::iterator &p)
 {
   info.decode(p);
 
+  num_completed_requests = info.completed_requests.size();
```

Comment:
size() is a constant-time function on standard maps, and num_completed_requests looks to just be an identical value. Please remove it and use size() instead.